WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/ShopinBit/1_Initial/WelcomeStep.cs

```diff
@@ -17,6 +16,8 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Buy.Workflows;
 /// </summary>
 public partial class WelcomeStep : WorkflowStep<BuyAnythingClient.Product?>
 {
+	public const string ServiceDescriptionUrl = "https://shopinbit.com/wasabiwelcome";
```

Comment:
The explanation link has changed since then. It is going to be here: https://wasabiwallet.io/buy-anything.html

https://github.com/zkSNACKs/WalletWasabi/issues/12446#issuecomment-1941397886